sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -28,26 +28,26 @@ export interface AdmNativeMessage {
 // @public
 export interface AdmNativeNotification {
     body?: string;
-    bodyLocArgs?: string[];
-    bodyLocKey?: string;
-    channelId?: string;
-    clickAction?: string;
+    body_loc_args?: string[];
+    body_loc_key?: string;
+    channel_id?: string;
+    click_action?: string;
     color?: string;
-    defaultSound?: boolean;
-    eventTime?: string;
+    default_sound?: boolean;
```

Comment:
this is because the native shape really does look like this and we don't want to editorialize?

Reply:
Correct, now keeping as is for example here https://firebase.google.com/docs/cloud-messaging/http-server-ref